src/Number.php

```diff
@@ -115,4 +115,36 @@ public static function fromFloat($floatingPoint)
 
         return new self(sprintf('%.8g', $floatingPoint));
     }
+
+    /**
+     * @return bool
+     */
+    public function isNegative()
+    {
+        return $this->number[0] === '-';
```

Comment:
Edge case: what happens if someone enters `-0` here? Will it have any effect at all?